packages/atomic/src/components/search/atomic-search-box/atomic-search-box.tsx

```diff
@@ -144,6 +155,23 @@ export class AtomicSearchBox {
    */
   @Prop({reflect: true}) public clearFilters = true;
 
+  /**
+   * Event that is emitted when redirection is triggered. If the default behaviour is not prevented, the search box will directly change the URL.
+   *
+   * @example
+   * <atomic-search-box
+   *   redirection-url="/search"
+   *   onRedirect={(e) => {
+   *     e.preventDefault();
+   *     // handle redirection
+   *   }}
+   * ></atomic-search-box>
```

Comment:
I think we should give an example that works without JSX.

Also, I would add code highlighting.

```suggestion
   * @example
   * ```html
   * <script>
   *   document.querySelector('atomic-search-box').addEventListener((e) => {
   *     e.preventDefault();
   *     // handle redirection
   *   });
   * </script>
   * ...
   * <atomic-search-box redirection-url="/search"></atomic-search-box>
   * ```
```

Reply:
Great!